python/services/partitionerservice/dmod/partitionerservice/service.py

```diff
@@ -450,32 +486,36 @@ async def manage_job_partitioning(self):
             while not self._job_util.lock_active_jobs(lock_id):
                 await asyncio.sleep(2)
 
-            for job in [j for j in self._job_util.get_all_active_jobs() if
-                        j.status_step == JobExecStep.AWAITING_PARTITIONING]:
+            for job in [j for j in self._job_util.get_all_active_jobs() if j.status_step == JobExecStep.AWAITING_PARTITIONING]:
```

Comment:
Any reason not to use a generator comprehension here instead of a list comprehension?

Reply:
Not really.   The only case for the list comprehension is that, this is what was done before.  I think I changed the whitespace handling to make it easier on my IDE, and just forgot to undo that before committing things.  There wasn't a functional change here, though.  But there's no reason not to do this, so I can push that in just a moment.